src/Symfony/Component/Form/Form.php

```diff
@@ -140,7 +140,7 @@ class Form implements \IteratorAggregate, FormInterface
     private $lockSetData = false;
 
     /**
-     * @var string|int|null
+     * @var string
      */
     private $name;
```

Comment:
I'd suggest initializing this to the empty string then

Reply:
Why? `$name` is initialized in the constructor with a `$config->getName()` value which is always a string.